CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Modified Traffic Monitor to poll over IPv6 as well as IPv4 and separate the availability statuses.
 - Modified Traffic Router to separate availability statuses between IPv4 and IPv6.
 - Modified Traffic Portal and Traffic Ops to accept IPv6 only servers.
+- TO endpoint `/api/x.y/servers/:id/deliveryservices` now optionally accepts `.json` on the end of the path.
```

Comment:
do we need this? seems a bit minor for the change log.

Reply:
I wasn't sure so I added it, I'll remove it.